am/config.py

```diff
@@ -103,3 +105,31 @@ def acoustic_model_config(
         config.phonology.future_length = phon_future_length
 
     return config
+
+
+def get_align_config_and_crp_for_corrected_applicator(crp: rasr.CommonRasrParameters, exit_penalty: float = 0.0):
```

Comment:
If you want you can also add return value types.

Reply:
done